workflow/scripts/calc-vaf.py

```
@@ -0,0 +1,309 @@
+    for variant in vcf_reader:
+        vaf_array = None
+
+        if calculate_from_ad:
+            vaf_array = calculate_vaf_from_ad(variant, vcf_reader.samples)
+        elif num_field and num_name and den_field and den_name:
+            vaf_array = calculate_vaf_from_fields(variant, num_field, num_name)
+
+        if vaf_array is not None:
+            try:
+                variant.set_format('VAF', vaf_array)
+            except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)